{
  "schema_version": "1.3.0",
  "id": "GHSA-w32h-m3r8-mq4c",
  "modified": "2022-12-18T06:31:09Z",
  "published": "2022-12-18T06:31:09Z",
  "aliases": [
    "CVE-2022-47516"
  ],
  "details": "An issue was discovered in the libsofia-sip fork in drachtio-server before 0.8.20. It allows remote attackers to cause a denial of service (daemon crash) via a crafted UDP message that leads to a failure of the libsofia-sip-ua/tport/tport.c self assertion.",
  "severity": [

  ],
  "affected": [

  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2022-47516"
    },
    {
      "type": "WEB",
      "url": "https://github.com/drachtio/drachtio-server/issues/244"
    },
    {
      "type": "WEB",
      "url": "https://github.com/davehorton/sofia-sip/commit/13b2a135287caa2d67ac6cd5155626821e25b377"
    }
  ],
  "database_specific": {
    "cwe_ids": [

    ],
    "severity": null,
    "github_reviewed": false
  }
}